Sources/Core/AWSClientRuntime/Sources/AWSClientRuntime/AWSClientConfigDefaultsProvider.swift

```diff
@@ -135,4 +135,19 @@ public class AWSClientConfigDefaultsProvider {
             rateLimitingMode: resolvedRateLimitingMode
         )
     }
+
```

Comment:
Function below is used to resolve a default value for account ID endpoint mode, if needed.